common/src/main/java/com/wynntils/features/user/AuraTimerOverlayFeature.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.RenderUtils;
+import com.wynntils.gui.render.VerticalAlignment;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.event.SubtitleSetTextEvent;
+import com.wynntils.mc.objects.CommonColors;
+import com.wynntils.mc.objects.CustomColor;
+import com.wynntils.utils.MathUtils;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class AuraTimerOverlayFeature extends UserFeature {
+    private static final float MAX_INTENSITY = 0.4f;
+    private static final int AURA_PROC_MS = 3200;
+    private static final String AURA_TITLE = "§4§n/!\\§7 Tower §6Aura";
+
+    @Config
+    public boolean vignetteOnAura = true;
+
+    @Config
+    public CustomColor vignetteColor = CommonColors.ORANGE;
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay auraTimerOverlay = new AuraTimerOverlay();
+
+    private long lastAuraProc = 0;
+
+    @SubscribeEvent
+    public void onSubtitle(SubtitleSetTextEvent event) {
+        if (!event.getComponent().getString().equals(AURA_TITLE)) return;
```

Comment:
... Here I am, once again, saying that this kind of logic belongs in the model, not the feature... :-)

Reply:
The actual logic is like 2 lines, I didn't really want to create a model for that

Reply:
I am looking forward to it :)